mava/wrappers/environment_loop_wrappers.py

```diff
@@ -229,7 +230,7 @@ def _compute_episode_statistics(
         # Collect the results and combine with counts.
         steps_per_second = episode_steps / (time.time() - start_time)
         mean_episode_return = np.mean(np.array(list(episode_returns.values())))
-
+        raw_sum_episode_return = np.sum(np.array(list(episode_returns.values())))
```

Comment:
Would `sum(episode_returns.values())` work here? Or do you need the type to be a numpy float?

Reply:
Same with the mean calculation.

Reply:
I will give it a try and back to you soon :)

Reply:
for the sum it works without numpy but for the mean we can make the equation instead of numpy if you find it really necessary to do so?
Should we remove np and replace it with sum and mean equation?

Reply:
This is minor, so whichever way you think is best :)

Reply:
I think we need to keep `np.sum` because it is more coeherent with `np.mean`, but is up to you!